spec/helpers/codelist_helper_spec.rb

```diff
@@ -48,6 +48,20 @@
           with_empty_item: false
         ).last).to eq(OpenStruct.new(name: "Zloty", code: "PLN"))
       end
+
+      it "does not add any items to the list with 'status: withdrawn'" do
+        list = helper.yaml_to_objects(
```

Comment:
It's a shame we can't overide the yaml here to make it clearer what is going on - I feel like I have to go and look at the yaml file to properly understand this. If we did that would it count as dependancy injection?

Reply:
I was wondering this too. When I originally wrote these tests way back when, I had a dummy YAML file for testing in the fixtures, but I was advised that using the "real" data was safer. I feel like using a dummy YAML for this instance might be clearer though?

Reply:
It would defo be clearer for me!

If `yaml_to_objects` took a `data` argument, I feel like we could pass in a nice clear piece of yaml in the test - not sure if it's worth the effort here though?

Reply:
Yeah I'm going to add a small fixture with some dummy data, I think it's best (and that's two votes for!)

Reply:
This is defintately better, I don't have to parse and understand that giant sectors file!

The one simple thing that would help me is to change the context to read:

"when there is only one active and two withdrawn items" it "filters out the withdrawn items"

or something like that, that explains the fixture without me needing to go look. then I would add an  expectation for the active item, somthing like:

`expect(list[1].status).to eq "active"`

that would make me super happy.